.github/CODEOWNERS

```diff
@@ -1,2 +1,3 @@
 # This repo is owned by the universal-publishing team, which is  administered by the FT Content Programme Team.
 * @Financial-Times/universal-publishing
+* @Financial-Times/platform-health
```

Comment:
Missing new line at the end of file.

And not required but a comment like `#This repo is supported by the platform-health team.` would be nice.